proc_square_refl.py

```diff
@@ -143,19 +143,28 @@ def filter_range(x): return array([j for j in x if
     d.setaxis('t', lambda x: x-time_zero).register_axis({'t':0})
     refl_blip_ranges -= time_zero
     pulse_range -= time_zero
-    fl.next('after setting t=0')
-    for j in range(2):
-        fl.plot(abs(d['ch',j]), linewidth=3, color='k',
-                label='ch %d abs'%(j+1), alpha=0.3)
```

Comment:
you had shown the data here, but not after all corrections -- I just move this to after you have done all your corrections